src/node/export.ts

```diff
@@ -119,11 +121,15 @@ async function copyFootage(footage: Footage[], projectDir: string) {
  *
  * @param zipPath The path of the directory to zip.
  */
-function zip(zipPath: string): Promise<void> {
+function zip(zipPath: string, projectName: string): Promise<void> {
   return new Promise((resolve, reject) => {
     const pathResolved = finalizePath(zipPath); // Normalize and resolve
 
-    const outputZipPath = `${pathResolved}.zip`;
+    // replace final part of path with project name to create zip with project name
+    const pathReplaced = path.join(path.dirname(pathResolved), projectName);
```

Comment:
what's the meaning of the `pathReplaced`? even the comment above is completely unclear..